action/protocol/poll/nativestaking.go

```diff
@@ -94,55 +94,60 @@ func (ns *NativeStaking) Votes() (*VoteTally, time.Time, error) {
 		Buckets:    make([]*types.Bucket, 0),
 	}
 	prevIndex := big.NewInt(0)
-	limit := big.NewInt(256)
+	limit := big.NewInt(160)
```

Comment:
Let's also change it from 256 to 160 after daytonaBlockHeight, so we can make sure past behavior is the same.